src/modules/configurator/components/ConfigProperty.tsx

```diff
@@ -0,0 +1,75 @@
+import { useConfigProperty } from "../hooks";
+
+interface PropertyProps {
+  index: number;
+  propertyName: "color" | "wheels";
+  modelName: "Audi RS5" | "Audi RS6" | "Audi e-tron GT" | undefined;
+  label: string | number;
+  name: string;
+  description: string;
+  price: number;
+}
+
+export function ConfigProperty({
+  index,
+  propertyName,
+  modelName,
+  label,
+  name,
+  description,
+  price,
+}: PropertyProps) {
+  const { handleOpenDropdown, propertyImgUrl, activePropIndex } =
+    useConfigProperty({
+      index,
+      propertyName,
+      modelName,
+      label,
+      name,
+      description,
+      price,
+    });
+
+  return (
+    <button
+      className="flex items-center sm:pr-40 lg:pr-48 3xl:pr-64"
+      onClick={handleOpenDropdown}
+    >
+      <div className="relative p-2 xs:p-3 sm:p-5">
+        {propertyImgUrl ? (
+          <img
+            src={propertyImgUrl}
+            alt={name}
+            className="w-10 sm:w-12 2xl:w-14 3xl:w-20 rounded-[50%]"
+          />
+        ) : (
+          <div className="w-12 2xl:w-14 h-12 2xl:h-14 bg-light-gray-background-color rounded-[50%]"></div>
```

Comment:
Should this be empty?

Reply:
Empty styling for when property image is loading. Changed it to self-closing tag to avoid confusion.